examples/routing/gamma_IRF.jl

```diff
@@ -0,0 +1,52 @@
+using ClimaRivers
+using JSON, Dates
+
+# build hillslope model
+shape = 1.5
+timescale = 1.0
+t_max_hs = 60.0
+hillslope = MizurouteHillslopeV1{Float64}(shape, timescale, t_max_hs)
```

Comment:
If you like, you can just use the defaults: `MizurouteHillslopeV1{Float64}()`, and same for the MizurouteChannelV1 struct